### PR TITLE
Branch

### PR DESCRIPTION
**Pull request by nightbrest**

### CODE REVIEW

File: pageobjects/bookStorePage.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    backToStoreBtn: '#addNewRecordButton',
+    searchBox: '#searchBox',
+      
+    changingBookTitle: (title) => `//span[contains(@id,"see-book-${title}")]`,
+    changingBookSubTitle: (subTitle) => `//label[@class="form-label" and .//text()="${subTitle}"]`,    
+    
+};
+
+class bookStorePage extends Page {
+    get bookTitl() { return $(selectors.bookTitl) };
```
Comment (HannaLuzko) [inline pageobjects/bookStorePage.js]: bookTitle

File: pageobjects/bookStorePage.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    
+    get bookInfo() { return $(selectors.bookInfo) };
+    
+    get booksISBN() { return $(selectors.booksISBN) };
+
+    get backToStoreBtn() { return $(selectors.backToStoreBtn) };
+
+    get searchBox() { return $(selectors.searchBox) };
+
+    BooksTitlesRow() {
```
Comment (HannaLuzko) [inline pageobjects/bookStorePage.js]: booksTitlesRow

File: step-definitions/bookStorePage.steps.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    await bookStorePage.bookStoreBtn.scrollIntoView();
+    await bookStorePage.bookStoreBtn.click();
+});
+
+Then(/^I should see book store with book items appeared$/, async () => {
+    expect(await bookStorePage.booksList.isDisplayed()).to.equal(true);
+});
+
+When(/^I click "Git Pocket Guide" book's title$/, async () => {
+    await bookStorePage.bookTitl.scrollIntoView();
```
Comment (HannaLuzko) [inline step-definitions/bookStorePage.steps.js]: bookTitle

File: step-definitions/bookStorePage.steps.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+});
+
+When(/^I tape keyword "(.*)" in the search field$/, async (searchWord) => {
+    await bookStorePage.searchBox.waitForDisplayed();
+    await bookStorePage.searchBox.setValue(searchWord);
+});
+
+Then(/^I should see as a search result following books$/, async (table) => {
+    const expected = [].concat(...table.rows());
+    const actual = await bookStorePage.BooksTitlesRow();
```
Comment (HannaLuzko) [inline step-definitions/bookStorePage.steps.js]: booksTitlesRow

**Review by nightbrest — COMMENTED** [review]
fixed mistakes

**Pull request closed without merging**